common/build/eslint-config-fluid/src/custom-rules/no-restricted-tags-imports.js

```diff
@@ -0,0 +1,91 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+const { TSDocParser } = require("@microsoft/tsdoc");
+
+module.exports = {
+	meta: {
+		type: "problem",
+		docs: {
+			description:
+				"This rule restricts imports from specified tags or non-public APIs. This to prevent accidental dependencies on internal, unstable or undocumented parts of the codebase.",
+			category: "Best Practices",
+		},
+		fixable: "code",
+		schema: [
+			{
+				type: "object",
+				properties: {
+					tags: {
+						type: "array",
+						items: { type: "string" },
+						uniqueItems: true,
+					},
+					exceptions: {
+						type: "array",
+						items: { type: "string" },
+						uniqueItems: true,
+					},
+				},
+				additionalProperties: false,
+			},
+		],
+		messages: {
+			importWithRestrictedTag: "Import with restricted tag found.",
+		},
+	},
+	create(context) {
+		const options = context.options[0] || {};
+		// Get restricted tags and throw error if not formatted correctly, ie: doesn't start with '@'
+		const restrictedTags = new Set(
+			(options.tags || []).map((tag) => {
+				if (!tag.startsWith("@")) {
+					context.report({
+						loc: { line: 1, column: 0 },
+						message: `Invalid tag format in rule configuration: '{${tag}}'. Tags should start with '@'.`,
+						data: { tag },
+					});
+					invalidTags = true;
```

Comment:
`invalidTags` is not declared anywhere. Is setting it like this enough to cause a validation issue? Are there docs about it that we can link here?

Reply:
I think this was some erroneous code I left in.